// This file was auto-generated by the veyron vdl tool.
// Source: errors.vdl

package vc

import (
	// VDL system imports
	"v.io/core/veyron2/context"
	"v.io/core/veyron2/i18n"
	"v.io/core/veyron2/verror"

	// VDL user imports
	"v.io/core/veyron2/security"
)

var (
	ErrNoBlessingsForPeer = verror.Register("v.io/core/veyron/runtimes/google/ipc/stream/vc.NoBlessingsForPeer", verror.NoRetry, "{1:}{2:} no blessing tagged for peer {3} in the BlessingStore. Rejected blessings: {4}")
)

func init() {
	i18n.Cat().SetWithBase(i18n.LangID("en"), i18n.MsgID(ErrNoBlessingsForPeer.ID), "{1:}{2:} no blessing tagged for peer {3} in the BlessingStore. Rejected blessings: {4}")
}

// NewErrNoBlessingsForPeer returns an error with the ErrNoBlessingsForPeer ID.
func NewErrNoBlessingsForPeer(ctx *context.T, peer []string, rejected []security.RejectedBlessing) error {
	return verror.New(ErrNoBlessingsForPeer, ctx, peer, rejected)
}
